packages/thumbnail/src/spectrum-thumbnail.css

```diff
@@ -12,353 +12,431 @@ governing permissions and limitations under the License.
-        --spectrum-thumbnail-l-border-color,
-        var(--spectrum-alias-border-color-translucent)
+    outline-offset: calc(
+        var(
+                --mod-thumbnail-border-width-selected,
+                var(--spectrum-thumbnail-border-width-selected)
+            ) -
+            var(
+                --mod-thumbnail-layer-border-width-inner,
+                var(--spectrum-thumbnail-layer-border-width-inner)
+            )
     );
-    --spectrum-thumbnail-border-color-key-focus: var(
-        --spectrum-thumbnail-l-border-color-key-focus,
-        var(--spectrum-alias-border-color-key-focus)
+    outline-width: var(
+        --mod-thumbnail-border-width-selected,
+        var(--spectrum-thumbnail-border-width-selected)
     );
-    --spectrum-thumbnail-darksquare-background-color: var(
-        --spectrum-thumbnail-l-darksquare-background-color,
-        var(--spectrum-alias-thumbnail-darksquare-background-color)
+}
+.spectrum-Thumbnail-layer-inner {
+    align-items: center;
+    block-size: calc(
+        var(--spectrum-thumbnail-size) -
+            var(
+                --mod-thumbnail-layer-border-width-inner,
+                var(--spectrum-thumbnail-layer-border-width-inner)
+            ) * 2
     );
-    --spectrum-thumbnail-border-radius: var(
-        --spectrum-thumbnail-l-border-radius,
-        var(--spectrum-alias-thumbnail-border-radius-small)
+    display: flex;
+    inline-size: calc(
+        var(--spectrum-thumbnail-size) -
+            var(
+                --mod-thumbnail-layer-border-width-inner,
+                var(--spectrum-thumbnail-layer-border-width-inner)
+            ) * 2
     );
-    --spectrum-thumbnail-width: var(
-        --spectrum-thumbnail-l-width,
-        var(--spectrum-global-dimension-size-700)
+    outline: solid;
+    outline-color: var(
+        --mod-thumbnail-layer-border-color-inner,
+        var(--spectrum-thumbnail-layer-border-color-inner)
     );
-    --spectrum-thumbnail-height: var(
-        --spectrum-thumbnail-l-height,
-        var(--spectrum-global-dimension-size-700)
+    outline-width: calc(
+        var(
+                --mod-thumbnail-layer-border-width-inner,
+                var(--spectrum-thumbnail-layer-border-width-inner)
+            ) -
+            var(
+                --mod-thumbnail-layer-border-width-outer,
+                var(--spectrum-thumbnail-layer-border-width-outer)
+            )
     );
 }
-::slotted(*) {
-    max-height: 100%;
-    max-width: 100%;
+.image-wrapper {
+    align-items: center;
+    block-size: 100%;
+    display: flex;
+    inline-size: 100%;
+    justify-content: center;
+}
+.image {
+    max-block-size: 100%;
+    max-inline-size: 100%;
+    position: relative;
     z-index: 1;
 }
-:host([cover]) ::slotted(*) {
-    height: 100%;
+:host([cover]) .image {
+    block-size: 100%;
+    inline-size: 100%;
     object-fit: cover;
     object-position: center;
-    width: 100%;
 }
 .background {
```

Comment:
This still being available points to a feature having been removed in you edits to `Thumbnail.ts`.